### ISSUE TITLE
Upgrade strangness?

### ISSUE THREAD

**Issue by jcoxdco** [tdulcet/AutoPrimeNet#2]
I had something happen as a result of my upgrade that I can't explain and it resulted in my LL DC not being submitted. Figured I'd post here in case it was some esoteric upgrade bug.
TLDR; somehow the entry of: resultsfile = egister-exponents made it into my prime.ini file (Not a typo. there was no "r". Presumably should have said "register-exponents" but even that doesn't make sense to me.

Here's how I upgraded primenet.exe to autoprimenet.exe
Downloaded autoprimenet
Copied it into the directory where primenet.exe was and deleted primenet.exe
Renamed local.ini to prime.ini
Ran through the setup again. (autoprimenet --setup)
Today I noticed that a completed LLDC hadn't been submitted. Opened the .ini file and egister-exponents jumped out pretty quickly. I changed the line to point to results.txt and everything now works fine. I just can't explain the strange entry. I haven't ever changed that line as I have always used the defaults for worktodo and results.

Perhaps it's nothing and if so, fantastic.

Any ideas?

**Comment by jcoxdco** [comment]
There is only one place in the code that has the text egister-exponents. It certainly isn't a bug related to upgrade. I must have typed register-exponents while a window was hidden or something. Closing as

**Comment by tdulcet** [comment]
> TLDR; somehow the entry of: resultsfile = egister-exponents made it into my prime.ini file (Not a typo. there was no "r".

I suspect that you attempted to use the `--register-exponents` option, but forgot one of the leading dashes, as in `-register-exponents`. This is the same as `-r egister-exponents` (with a space) using the `-r/--results-file` option and would thus change the default results file from `results.txt` to `egister-exponents`.

Anyway, it looks like you figured how to fix this. Please feel free to reopen the issue if you have further questions.
